subsys/usb/usb_work_q.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ * Copyright (c) 2018 Intel Corporation
+ * Copyright (c) 2016 Wind River Systems, Inc.
```

Comment:
Should those be there if this is a new file? Just asking.

Reply:
It's basically a copy of `kernel/system_work_q.c` with a few identifiers changed, so I preserved those copyrights.